src/openfluid/base/RunContextManager.cpp

```diff
@@ -470,5 +471,32 @@ void RunContextManager::processWareParams(openfluid::ware::WareParams_t& Params)
 }
 
 
+// =====================================================================
+// =====================================================================
+
+
+void RunContextManager::computeDefaultBufferSize(const float Duration, const float DeltaT)
```

Comment:
To move into openfluid::machine::computeValuesBuffersDefaultSize(Duration,DT) in Engine.hpp file